packages/server/tsconfig.json

```diff
@@ -114,8 +114,10 @@
     "test/**/*",
     "app.ts",
     "otel.ts",
+    "bootstrap.ts",
+    "knexfile.ts",
     "bootstrap.js",
-    "knexfile.js"
+    "knexfile.ts"
```

Comment:
bootstrap.js is the only one remaining, but that's cause it doesn't support ESM - it relies on synchronous requires and needs an alternate async variant in ESM